src/engraving/dom/vibrato.cpp

```diff
@@ -29,6 +29,8 @@
 
 #include "score.h"
 #include "system.h"
+#include "chord.h"
```

Comment:
seems this include is not needed

Reply:
Feel free to remove it in https://github.com/musescore/MuseScore/pull/25425 in that case

Reply:
OK, will do